lib/simp/cli/config/items/data/chronyd_link_to_ntpservers.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../list_item'
+require_relative '../../utils'
+require_relative 'cli_network_gateway'
+
+module Simp; end
+class Simp::Cli; end
+module Simp::Cli::Config
+  class Item::ChronydNTPServersDefault < Item
```

Comment:
Class should be named for hieradata setting and name of the file should match the class name.  I'll fix.